pkg/operator2/deployment.go

```diff
@@ -121,6 +122,7 @@ func defaultDeployment(
 								},
 							},
 							VolumeMounts:             mounts,
+							Env:                      proxyConfigToEnvVars(proxyConfig),
```

Comment:
How is the stuff for the operator itself going?  I see this is meant for the OAuth server.

Reply:
That should be handled from CVO, given that the operator itself can be considered its operand:
https://github.com/openshift/cluster-version-operator/pull/224